sp_HumanEvents/sp_HumanEventsBlockViewer.sql

```diff
@@ -841,7 +930,7 @@ WHERE b.n = 1
 AND   (b.contentious_object = @object_name
        OR @object_name IS NULL)
 ORDER BY
-    b.event_time DESC,
+    b.sort_order,
```

Comment:
`event_time` of a blocked process report doesn't make much sense  when looking at processes that take part within the same deadlock monitor loop.
The calculated `sort_order` is used to make the blocking_tree make sense (i.e. lead blockers up front)